services/collectd/config_test.go

```diff
@@ -28,5 +29,7 @@ typesdb = "yyy"
 		t.Fatalf("unexpected database: %s", c.Database)
 	} else if c.TypesDB != "yyy" {
 		t.Fatalf("unexpected types db: %s", c.TypesDB)
+	} else if len(c.TypesDBDirs) != 2 {
```

Comment:
There are often other files in the default types.db directory (default postgres queries, java files, ...)